content/influxdb/v2.0/reference/internals/system-buckets.md

```diff
@@ -0,0 +1,84 @@
+---
+title: InfluxDB system buckets
+description: >
+  InfluxDB system buckets contain time series data used by and generated from the
+  InfluxDB monitoring and alerting system and the task engine.
+menu:
+  influxdb_2_0_ref:
+    name: System buckets
+    parent: InfluxDB internals
+weight: 101
+influxdb/v2.0/tags: [buckets]
+related:
+  - /influxdb/v2.0/monitor-alert/
+  - /influxdb/v2.0/process-data/
+  - /influxdb/v2.0/reference/flux/stdlib/monitor/check/
+  - /influxdb/v2.0/reference/flux/stdlib/monitor/deadman/
+  - /influxdb/v2.0/reference/flux/stdlib/monitor/from/
+  - /influxdb/v2.0/reference/flux/stdlib/monitor/logs/
+  - /influxdb/v2.0/reference/flux/stdlib/monitor/notify/
+---
+
+InfluxDB system buckets contain time series data used by and generated from the
+InfluxDB monitoring and alerting system and the task engine.
+
+The following system buckets exist in each InfluxDB organization:
+
+- [_monitoring](#_monitoring-system-bucket)
+- [_tasks](#_tasks-system-bucket)
+
+## _monitoring system bucket
+The `_monitoring` system bucket stores data generated from and used by InfluxDB
+[checks and notifications](/influxdb/v2.0/monitor-alert/).
+
+### _monitoring bucket schema
+
```

Comment:
In case some lands on "internals" before key concepts, we could explicitly state:
The following schema references InfluxDB data elements. For more information, see [InfluxDB key concepts](/influxdb/cloud/reference/key-concepts/data-elements/).